Source/WebCore/svg/SVGStringList.cpp

```diff
@@ -63,6 +63,38 @@ bool SVGStringList::parse(StringView data, UChar delimiter)
     });
 }
 
+bool SVGStringList::resetCommaSeparatedTokens(StringView data)
```

Comment:
I asked @nikolaszimmermann last week to take a look at this, and he suggested refactoring `parse()` to support the case where we keep empty records, and use that instead, instead of largely duplicating it.

So what this might look like is,

```c++
if (buffer.position() == start) {
  if (mode == ParseModeKeepEmptyFields)
    m_items.append(String( { start, buffer.position() }));
  break;
}
```
Or something like that -- I think continuing to use the `parse()` name could make the intent of the call more clear.